model/item.cc

```diff
@@ -118,6 +118,48 @@ namespace minsky
     {
       return abs(x0-r.x(x1,y1))<d && abs(y0-r.y(x1,y1))<d;
     }
+}
+    
+   std::pair<double,std::pair<float,float>> Item::rotatedPoints() const
```

Comment:
You can use the Point datatype (defined in geometry.h) for the second part of this pair.